test/test_transforms.py

```diff
@@ -130,6 +131,20 @@ def test_ten_crop(self):
                 assert len(results) == 10
                 assert expected_output == results
 
+    def test_randomresized_params(self):
+        height = random.randint(24, 32) * 2
+        width = random.randint(24, 32) * 2
+        img = torch.ones(3, height, width)
+        to_pil_image = transforms.ToPILImage()
+        img = to_pil_image(img)
+        size = 100
+        aspect_ratio_range = (0.75, 1.3)
+        scale_range = (0.08, 1)
+        randresizecrop = transforms.RandomResizedCrop(size, scale_range, aspect_ratio_range)
+        _, _, h, w = randresizecrop.get_params(img, scale_range, aspect_ratio_range)
```

Comment:
Can you run this multiple times in a for loop, asserting that the aspect ratio is within bounds?

Reply:
@fmassa done !!

Reply:
@fmassa  yeah fixed it !